backend/src/main/java/com/example/backend/Repositories/BankAccountRepository.java

```diff
@@ -13,5 +14,5 @@ public interface BankAccountRepository extends JpaRepository<BankAccount, Long>
 
 
     @Query("select b from BankAccount b where b.user = ?1")
-    List<BankAccount> findBankAccountsByUser(User user);
+    ArrayList<BankAccount> findBankAccountsByUser(User user);
```

Comment:
Jak cos wydaje mi sie ze nie zadziala z List

Reply:
w sumie to tego wcale nie musi być